validate.go

```diff
@@ -208,6 +211,23 @@ func checkLinux(spec rspec.Spec) {
 			logrus.Fatalf("namespace %v is invalid.", spec.Linux.Namespaces[index])
 		} else if spec.Linux.Namespaces[index].Type == rspec.UTSNamespace {
 			utsExists = true
```

Comment:
On 07/06/2016 11:39 AM, W. Trevor King wrote:
> I think we want:
> 
> |else if len(spec.Linux.Namespaces[index].Path) == 0 { continue } |
> 
> as the first block after the validity check. That way the logic <https://github.com/opencontainers/ocitools/pull/124#discussion_r68693222> also applies to this UTS case, and we don't have to duplicate |Path| checks for each namespace below.
> 

Yes, it really is. will fix.

-- 
Ma Shimiao
Development Dept.I
Nanjing Fujitsu Nanda Software Tech. Co., Ltd.(FNST)